requirements/constraints.txt

```diff
@@ -8,30 +8,26 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 
-# Already in python3 standard library
-futures; python_version == "2.7"
-
 # Test dependencies
 pylint<2.14
 pycodestyle==2.8.0
 factory-boy==3.2.1
-django-countries==7.3.2
+django-countries==7.4.2
 
 # Keep same platform version
 click>=8.0,<9.0
 celery>=5.2.2,<6.0.0
-django<=3.2.15
+django<4.0
 djangorestframework==3.12.4
 django-filter==21.1
 django-model-utils==4.2.0
 django-oauth-toolkit<=1.3.2
-django-waffle==2.4.1
-edx-api-doc-tools<=1.6.0
-edx-drf-extensions==8.0.1
+django-waffle==3.0.0
+edx-api-doc-tools==1.6.0
+edx-drf-extensions==8.3.1
 edx-opaque-keys==2.3.0
-edx-proctoring==4.10.2
+edx-proctoring>=2.0.1
 jsonfield==3.1.0
-django-countries==7.3.2
-edx-django-utils==4.6.0
+edx-django-utils>=5.1.0
 edx-when==2.3.0
-newrelic==7.10.0.175
+newrelic==8.2.1
```

Comment:
newrelic>8.2.1 causes problems?
I ask because in edx-platform in open-release/nutmeg.master they have newrelic==7.10.0.175 but in master they allow newrelic==8.5.0. Do you have a reason to set `newrelic==8.2.1`?

Reply:
That's the version used by edx-platform @ olive. Actually, I don't even know why it was set to `newrelic==7.10.0.175` before since in nutmeg it wasn't pinned either. I'll remove the constraint

Reply:
You are right; I don't know why I comment about nutmeg when it's Olive 🙈
Note: I think we pinned before to avoid a big upgrade we didn't want.